packages/terra-arrange/src/Arrange.module.scss

```diff
@@ -37,3 +37,9 @@
     align-self: flex-start;
   }
 }
+
+.fill-block {
+  .title-block {
```

Comment:
this class seems to be required by terra-section-header and not required by terra-arrange. 
if css variable is added then it needs value on other themes as well !!